apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -48,11 +72,24 @@ class LibraryCreationDialog extends React.Component {
   };
 
   publish = () => {
+    let formElements = document.getElementById('selectFunction').elements;
+    let selectedFunctionList = [];
+    let libraryDescription = '';
+    [...formElements].forEach(element => {
+      if (element.type === 'checkbox' && element.checked) {
+        selectedFunctionList.push(this.state.sourceFunctionList[element.value]);
+      }
+      if (element.type === 'textarea') {
```

Comment:
NIT: (I don't feel too strongly about this) This seems like it would be more maintainable as a check by class name rather than type. i.e. it's not too unlikely that we'd add another textarea somewhere.